### PR TITLE
made edits to strings.md.

### LINKED ISSUE #5001: [Edit] JavaScript Strings
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

We would like to edit the `Strings` concept entry under JavaScript. The entry lives in `content/javascript/concepts/strings/strings.md`.

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

Please:
1) Add a `Codebyte Example` section that provides a codebyte example demonstrating the concept in use
2) Ensure that the codebyte example is different than all the other examples in the entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by andrewjliao414**
### Description

Simply edited strings.md. Made some grammatical changes, added Codebytes, and edited one of the codeblocks. The motivation was to complete the project in the Learn Git & GitHub course.

### Issue Solved

Closes #5001 
### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x ] All writings are my own.
- [x ] My entry follows the Codecademy Docs style guide.
- [x ] My changes generate no new warnings.
- [x ] I have performed a self-review of my own writing and code.
- [x ] I have checked my entry and corrected any misspellings.
- [x ] I have made corresponding changes to the documentation if needed.
- [x ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x ] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/javascript/concepts/strings/strings.md [hunk 1]
```diff
@@ -21,13 +22,12 @@ In JavaScript, multiple strings can be concatenated or joined together using the
...
 
-```js
+```codebyte/javascript
 let service = 'credit card';
 let month = 'May 30th';
 let displayText = 'Your ' + service + ' bill is due on ' + month + '.';
 
 console.log(displayText);
-// Output: Your credit card bill is due on May 30th.
 ```
```
Comment (mamtawardhani) [inline content/javascript/concepts/strings/strings.md:31]: You don't need to replace the existing code with a new one. You have to add a `##Codebyte Example` block, which is supposed to be at the end of the entry and should cover all the three topics above which are: Concatenation, interpolation, and escaping characters

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @andrewjliao414 thank you for your contribution  to Codecademy Docs! The entry looks really nice! 😃 

I have requested one change here, which is don't change any of the existing codes. You are supposed to add a new ##Codebyte Example block at the end of the entry which should cover all the three sub-topics covered in this PR: Concatenation, Interpolation and Escaping Characters.

Please review and modify this at your earliest convenience. Thank you! 😃

File: content/javascript/concepts/strings/strings.md (hunk 1 above)
Comment (andrewjliao414) [reply content/javascript/concepts/strings/strings.md:31]: Okay. I have made some edits based on your feedback. Take a look. Strings.md is attached to this comment.

Comment (andrewjliao414) [reply content/javascript/concepts/strings/strings.md:31]: If you don't see my updated strings.md let me know.

**Review by andrewjliao414 — COMMENTED** [review]
[strings.md](https://github.com/user-attachments/files/16574127/strings.md)
Updated strings.md.

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @andrewjliao414, the edits so far look nice! Could you also add a codebyte example block at the end using the `##Codebyte Example`.
In this, add one example each for all the three concepts in this entry.
Thank you!

**Comment by andrewjliao414** [comment]
[strings.md](https://github.com/user-attachments/files/16591496/strings.md)
Updated strings.md.

**Comment by mamtawardhani** [comment]
> [strings.md](https://github.com/user-attachments/files/16591496/strings.md) Updated strings.md.

Hey @andrewjliao414 have you added the codebyte example block at the end? It is not visible to me.

**Review by mamtawardhani — APPROVED** [review]
Thank you for making the changes @andrewjliao414 😄 

The entry looks good for a second round of review! 🚀

**Comment by andrewjliao414** [comment]
@mamtawardhani any update on when it will be reviewed for a second time?

**Comment by Sriparno08** [comment]
Hey @andrewjliao414, the second review will likely be done in a week.

**Review by Sriparno08 — APPROVED** [review]
Looks good, @andrewjliao414!

**Pull request merged**